apps/dashboard/app/apps/ood_app_link.rb

```diff
@@ -44,15 +44,20 @@ def categorize(category: nil, subcategory: nil)
   class LinkCategorizer < SimpleDelegator
     attr_reader :category, :subcategory
 
-    def initialize(link, category: nil, subcategory: nil)
+    def initialize(link, category: nil, subcategory: nil, show_in_menu: false)
```

Comment:
If everything else is defaulting to `''` this likely should to.

```suggestion
    def initialize(link, category: '', subcategory: '', show_in_menu: false)
```